lib/cisco_node_utils/cmd_ref/bgp_neighbor.yaml

```diff
@@ -8,7 +8,7 @@ _template:
   context:
     - "router bgp <asnum>"
     - "(?)vrf <vrf>"
-    - "neighbor <nbr>"
+    - "(?)neighbor <nbr>"
```

Comment:
Please do double-check and follow up with me if there are issues - would prefer to separate this change from the doc updates if possible.